readme.md

```diff
@@ -174,6 +174,45 @@ Next, create a `MyLambda.swift` and implement your Lambda. Note that the file ca
 
  Beyond the small cognitive complexity of using the `EventLoopFuture` based APIs, note these APIs should be used with extra care. An `EventLoopLambdaHandler` will execute the user code on the same `EventLoop` (thread) as the library, making processing faster but requiring the user code to never call blocking APIs as it might prevent the underlying process from functioning.
 
+## Testing locally
+
+Before to deploy your code to AWS Lambda, you can test it locally with the command:
+
+```sh
+LOCAL_LAMBDA_SERVER_ENABLED=true swift run
+```
+
+This starts a local HTTP endpoint listening on port TCP 7000.  Your can invoke your code by sending an HTTP POST command to `http://127.0.0.1:7000/invoke`.
+
+For example:
+
+```sh
+curl -v --header "Content-Type:\ application/json" --data @events/create-session.json http://127.0.0.1:7000/invoke
```

Comment:
Would be nice to have a line of explanation about `@events/create-session.json` just to mention it's reading from a file at that path 🙂, and if the JSON needs to be in any specific format for the lambda to be able to work with it.
Perhaps even a simple example of such JSON if it needs to be in a format.

Reply:
OK added with a link to a sample payload in the API Gateway documentation